quality/data_availability_dlt.sql

```diff
@@ -84,7 +85,14 @@ CREATE OR REFRESH LIVE TABLE data_availability
         FROM indicator.health_expenditure
         WHERE oop_per_capita_usd is not null
         GROUP BY 1
-    )
+    ),
+
+    subnat_pres as (
+        SELECT country_name, min(year) as subnat_pres_earliest_year, max(year) as subnat_pres_latest_year
```

Comment:
Let's rename `subnat_pres_earliest_year` to `boost_subnat_earliest_year` & ditto for latest, as there are other datasets involved in the data availability table.